tour/basic-read-write.md

```diff
@@ -1,5 +1,61 @@
 ---
-title: Basic Reading & Writing
+title: Writing and Reading
 ---
+Accumulo is a big data key/value store.  Writing data to Accumulo is flexible and fast.  Like any database, Accumulo stores
+data in tables and rows.  Each row in an Accumulo table can hold many key/value pairs.  
 
-Talk about reading and writing.
+Copy and paste the code below into the _exercise_  method.
+```java
+        // 1. Start by connecting to Mini Accumulo as the root user and create a table called "superheroes".
+        Connector conn = mac.getConnector("root", "tourguide");
+        conn.tableOperations().create("superheroes");
+
+        // 2. Create a Mutation object to write to a row
+        Mutation mutation = new Mutation("hero023948092");
+        // A Mutation is an object that holds all changes to a row in a table.  Each row has a unique row ID.
+
+        // 3. Create key/value pairs for Batman.  Put them in the "HeroAttribute" family.
+        mutation.put("HeroAttribute","name", "Batman");
+        mutation.put("HeroAttribute","real-name", "Bruce Wayne");
+        mutation.put("HeroAttribute","wearsCape?", "true");
+        mutation.put("HeroAttribute","flies?","false");
+
+        // 4. Create a BatchWriter to the superhero table and add your mutation to it.  Try w/ resources will close for us.
+        try(BatchWriter writer = conn.createBatchWriter("superheroes", new BatchWriterConfig())) {
+            writer.addMutation(mutation);
+        } catch(TableNotFoundException | MutationsRejectedException e) {
```

Comment:
I think it would be nice to make the exercise method throw Exception and drop this catch block.  Doing this makes this first bit of code shorter someone sees shorter, which I think is important.

Reply:
It does throw Exception, mainly because of the Connector.  For some reason I felt compelled to have a catch.  I guess it just feels wrong to have a try without a catch haha.  But it would greatly shorten the code.